src/lib/components/Button/Button.scss

```diff
@@ -0,0 +1,155 @@
+@use "sass:map";
+
+$colors: (
+  "primary": #0051ad,
+  "primary-dark": #063c7a,
+  "primary-darker": #00284d,
+  "disabled": #c9c9c9,
+  "disabled-dark": #adadad,
+  "base-lightest": #f2f1f0,
+  "base-light": #a9aeb1,
+  "base-dark": #565c65,
+  "base-darker": #3d4551,
+  "gray-05": #f0f0f0,
+  "primary-lightest": #d1e9ff,
+  "text-only-hover": rgba(6, 60, 122, 16%),
+  "text-only-active": rgba(6, 60, 122, 30%),
+  "outline-inverse": #dcdee0,
+);
```

Comment:
My site title PR defined the current color palette as [global variables](https://github.com/la-ldaf/ldaf-site/pull/50/files#diff-0a91e0c8de5f30d8b403617ebcd0339692b61f0ce8b5cb4106b5214ba6058842). That covers everything here except for the last 3 (and `gray-05`, although `$grayscale-05` is extremely close if that's an acceptable substitution, cc @getpunched )

If we like that approach, I can make a smaller PR to just get that in or do follow-up on this once #50 gets in.

Reply:
A follow-up PR to add that would be great! I've added a TODO comment indicating that we should do that.